deploy/nginx/Dockerfile

```diff
@@ -1,6 +0,0 @@
-FROM nginx:latest
```

Comment:
Not sure why removing this. Hope I did not confuse u to do that :) We can discuss on standup.